lnwallet/commitment.go

```diff
@@ -807,7 +976,11 @@ func (cb *CommitmentBuilder) createUnsignedCommitmentTx(ourBalance,
 		if err != nil {
 			return nil, err
 		}
-		cltvs = append(cltvs, htlc.Timeout) // nolint:makezero
+
+		// We want to add the CLTV and HTLC index to their respective
+		// slices, even if we already pre-allocated them.
+		cltvs = append(cltvs, htlc.Timeout)               //nolint
```

Comment:
Unfortunately we need it. Otherwise it complains about `make([]uint32, len(commitTx.TxOut))` (missing `0` secondary arg) and `append()` being used together (which often is wrong but in this specific context is what we want to do).
Comment above was supposed to explain this, but can be more clear if that's imprecise.